src/ActionModal/utils/ActionManager.js

```diff
@@ -67,13 +67,14 @@ export default class ActionManager {
     return gasEstimate
   }
 
-  async send(memo, txMetaData) {
+  async send(memo, txMetaData, subData) {
     this.readyCheck()
 
     const { gasEstimate, gasPrice, submitType, password } = txMetaData
     const signer = getSigner(config, submitType, {
       address: this.context.userAddress,
-      password
+      password,
+      subData
```

Comment:
What is 'subdata'?

Reply:
txMetaData was taken so it is an alternative. I tried avoiding a name that would suggest it is only for validator monikers in case we want to pass more data through in the future. Would just `metaData` be better?

Reply:
would it make sense to merge it with txMetaData? Or do you think splitting the 2 data blobs makes more sense?